test/e2e/tests/send-eth.spec.js

```diff
@@ -341,13 +342,15 @@ describe('Send ETH from dapp using advanced gas controls', function () {
         await priorityFeeInput.fill('1');
 
         await driver.clickElement({ text: 'Save', tag: 'button' });
+
         await driver.waitForSelector({
-          css: '.transaction-detail-item:nth-of-type(1) h6:nth-of-type(2)',
-          text: '0.04503836 ETH',
+          css: '.currency-display-component__text',
+          text: '0.05684869',
```

Comment:
Is there a reason for the different amount?

Reply:
Not sure, but tests are not passing with old value